plot_krun_results.py

```diff
@@ -448,9 +464,9 @@ def create_cli_parser():
     plt.close()  # avoid extra blank window
     if not options.interactive:
         print 'Saving results to: %s' % options.outfile
-    print (('Charting with sliding window size: %d, confidence level: %d ' +
+    print (('Charting with sliding window size: %d, zoom: %s, confidence level: %d ' +
             'bootstrap samples: %d and percentiles: %s') %
```

Comment:
This is a bit pedantic, but just to be clear can we say something like "xlimits: %s" rather than "zoom"?